### PR TITLE
No more cmsStage etc. scripts

### PR DESCRIPTION
**Pull request by hroskes**
https://hypernews.cern.ch/HyperNews/CMS/get/tif-alignment/465.html
Also rebased on #11195 which would have generated a merge conflict

### CODE REVIEW

**Comment by diguida** [comment]
please test

File: Alignment/OfflineValidation/python/TkAlAllInOneTool/zMuMuValidationTemplates.py [hunk 1]
```diff
@@ -23,11 +23,11 @@
 ########### standard includes ##############################
 process.load("Configuration.StandardSequences..oO[magneticField]Oo._cff")
 process.load("Configuration.StandardSequences.Reconstruction_cff")
-process.load("Configuration.Geometry.GeometryDB_cff")
+process.load("Configuration.Geometry.GeometryRecoDB_cff")
 
 
 ########### DATABASE conditions ############################
-process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
+process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff")
```
Comment (diguida) [inline Alignment/OfflineValidation/python/TkAlAllInOneTool/zMuMuValidationTemplates.py]: @hroskes in 76X and 75X V2 is default, in 74X you should keep the additional part in the name.

File: Alignment/OfflineValidation/test/zmumuValidate.ini [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+#zmumu Run2015B_ALCARECO_251604-251642 : alignment_hp1401
+#zmumu Run2015B_ALCARECO_251604-251642 : alignment_mp1819
+#zmumu Run2015B_ALCARECO_251604-251642 : alignment_mp1820
+#zmumu Run2015B_ALCARECO_251604-251642 : alignment_mp1823p4
+#zmumu Run2015B_ALCARECO_251604-251642 : alignment_mp1826p1
+#zmumu Run2015B_ALCARECO_251604-251642 : alignment_mp1826p2
+zmumu Run2015B_ALCARECO_251604-251642 : alignment_mp1834p0
+zmumu Run2015B_ALCARECO_251604-251642 : alignment_mp1834p1
+#zmumu Run2015B_ALCARECO_251604-251642 : alignment_mp1835p0
+#zmumu Run2015B_ALCARECO_251604-251642 : alignment_mp1835p1
```
Comment (diguida) [inline Alignment/OfflineValidation/test/zmumuValidate.ini]: I do not know it is good to have this "legacy" ini file in release. As you can see, there are many old conditions.

**Comment by hroskes** [comment]
Ok, I didn't really mean to deal with all this, I was just rebasing to avoid conflicts.  I'll talk to @usarica

**Comment by hroskes** [comment]
After consulting with Ulascan I removed the large .ini file, I just left one example so people have something to copy the syntax.
@diguida @usarica does this look ok to you?

**Comment by usarica** [comment]
It looks fine to me. The reason why I kept that file was to provide an example of what options can be used to run the Zmumu validation tool. testValidate.ini works equally well to keep the exemplary configuration.

**Comment by diguida** [comment]
please test
Comments addressed, thanks. Please back-port them to 75X and 74X PR, too (in the latter, do not modify the conddb version).
Concerning the "legacy" ini file for ZMuMu: I suggest to keep track on this in the alignment repository.

**Comment by mmusich** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**